src/ert/analysis/_es_update.py

```diff
@@ -751,26 +751,23 @@ def analysis_IES(
             AnalysisStatusEvent(msg="Loading observations and responses..")
         )
 
-        # Load responses, and observations
-        try:
+        (
+            S,
             (
-                S,
-                (
-                    observation_values,
-                    observation_errors,
-                    update_snapshot,
-                ),
-            ) = _load_observations_and_responses(
-                source_fs,
-                alpha,
-                std_cutoff,
-                global_scaling,
-                iens_active_index,
-                update_step.observation_config(),
-                misfit_preprocessor,
-            )
-        except IndexError as e:
```

Comment:
Why have you decided to remove try-except here?

Reply:
Because I want the function itself to raise errors and not users having to wrap it in try-except themselves.